### PR TITLE
Improvement for issue #3043

### PR DESCRIPTION
**Pull request by rsocol**

### CODE REVIEW

File: sp_BlitzIndex.sql [hunk 1]
```diff
@@ -2960,7 +2960,8 @@ BEGIN
 						LEFT OUTER JOIN ' + QUOTENAME(@DatabaseName) + N'.sys.partition_range_values prvs ON prvs.function_id = pf.function_id AND prvs.boundary_id = p.partition_number - 1
 						LEFT OUTER JOIN ' + QUOTENAME(@DatabaseName) + N'.sys.partition_range_values prve ON prve.function_id = pf.function_id AND prve.boundary_id = p.partition_number ' ELSE N' ' END 
 						+ N' LEFT OUTER JOIN ' + QUOTENAME(@DatabaseName) + N'.sys.column_store_segments seg ON p.partition_id = seg.partition_id AND ic.index_column_id = seg.column_id AND rg.row_group_id = seg.segment_id
-						WHERE rg.object_id = @ObjectID' 
+						WHERE rg.object_id = @ObjectID
+						AND c.name IN ( ' + REPLACE(REPLACE(@ColumnList,'[',''''),']','''') + N')'
```
Comment (BrentOzar) [inline sp_BlitzIndex.sql:2964]: Thanks for the pull request, but this won't work - I'll leave a repro script in the issue notes.

Comment (rsocol) [reply sp_BlitzIndex.sql]: Is it ok now?

**Comment by rsocol** [comment]
@BrentOzar, did you have a chance to look at the commit done on 30 November? (https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/pull/3044/commits/9d24e32d0ad6814ee6adc2c3cc3746c349e53d62)

**Comment by BrentOzar** [comment]
No, I'm on vacation in Mexico with my family. I'll get to it in Jan when I'm back though. Thanks!

---
Tiny glass keyboard
Typos flowing like rivers
As winter snow thaws

**Review by BrentOzar — APPROVED** [review]
Thanks for the pull request. Looks good. Merging into the dev branch, will be in the next release with credit to you in the release notes.

**Pull request merged**